### PR TITLE
Stop feed button

### PR DESCRIPTION
**Pull request by lizstarin**

### CODE REVIEW

File: public/assets/js/app/controllers/feedsController.js [hunk 1]
```diff
@@ -28,6 +28,32 @@ function FeedsCtrl($scope, $rootScope, $feedDataPaths, $feedsService, $location,
...
 
+  function postStopFeed(feedid) {
+    if ($authService.hasRole('super-admin')) {
+      $authService.getUser(function (user) {
+        console.log("feed stop requested by " + user.userName);
+        $http.post('/db/feeds/' + feedid + '/stop', {user_name: user.userName}).
+        then(
+          function(result) {
+            if(result.data.length === 0) {
+              $rootScope.pageHeader.alert = "Feed could not be stopped.";
```
Comment (gws) [inline public/assets/js/app/controllers/feedsController.js:39]: ominous

**Review by gws — COMMENTED** [review]
@lizstarin LGTM! I just wanted to check that the CSS that's missing on this branch compared to staging is intentional: https://github.com/votinginfoproject/Metis/compare/staging..stop-feed-button

File: public/assets/js/app/controllers/feedsController.js (hunk 1 above)
Comment (lizstarin) [reply public/assets/js/app/controllers/feedsController.js:39]: Good catch—that's not intentional.

Comment (lizstarin) [reply public/assets/js/app/controllers/feedsController.js:39]: (although I shouldn't have pushed straight to staging—more git badness])

**Review by gws — APPROVED** [review]

**Pull request merged**